apps/docs/docs/state/faq/faq.md

```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_label: 'FAQ'
-sidebar_position: 5
+sidebar_position: 60
```

Comment:
Why are you changing the sidebar position to x10 number?
Did you mean 6?

Reply:
This is the 5th item from the end.